Makefile

```diff
@@ -232,6 +232,7 @@ ci-tunnel: fix-macos-vendor
 
 .PHONY: ci-clean
 ci-clean:
+	$(shell podman ps --external --format "{{.Command}} {{.ID}}" | grep buildah | cut -d " " -f 2 | xargs podman rm -f > /dev/null)
```

Comment:
I sersiously wish this command could be prettier - I went through many iterations before settling here. If `podman ps --filter` supported COMMAND, this would be trivial, but it doesn't, and `/dev/null` is required because `make` does weird things with STDOUT on `$(shell)` stuff (you get errors like `<Deleted Container SHA>: command not found`) :/

Reply:
We could shorten it a bit with command substitution.

podman rm -f $(podman ps | grep buildah | awk '{print $1}')

Or, could we add a name to the build container so we can use the rm filter?